lib/src/signed_video_openssl.c

```diff
@@ -319,48 +320,58 @@ openssl_hash_data(void *handle, const uint8_t *data, size_t data_size, uint8_t *
 
 /* Initializes EVP_MD_CTX in |handle| with |hash_algo.type|. */
```

Comment:
Update description from "Initializes EVP_MD_CTX ..." to "Initializes a EVP_MD_CTX ..."